runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -1348,21 +1770,15 @@ TR::Register *J9::Power::TreeEvaluator::irdbariEvaluator(TR::Node *node, TR::Cod
 
    cg->machine()->setLinkRegisterKilled(true);
 
-   return objReg;
-#else
-   return NULL;
+   return objReg;   
 #endif
    }
 
-TR::Register *J9::Power::TreeEvaluator::ardbarEvaluator(TR::Node *node, TR::CodeGenerator *cg)
-   {
-   return TR::TreeEvaluator::ardbariEvaluator(node, cg);
-   }
-
-TR::Register *J9::Power::TreeEvaluator::ardbariEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+TR::Register *aGenerateSoftwareReadBarrier(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
Maybe a better name would be `uncompressedObjLoadWithReadBarrier`. On x they use `performHeapLoadWithReadBarrier` but we can't use that because we have two different versions for compressed and uncompressed objects.